src/symbols/collections.rs

```diff
@@ -1,64 +1,129 @@
 // Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
 
+#![allow(dead_code)]
```

Comment:
Remove?

Reply:
It's necessary because some methods are not used in the macro depenending on the implementation (ex. it might be used in the HashMap one, but not the IndexMap one).